typings/index.d.ts

```diff
@@ -30,20 +30,26 @@ export class GiveawaysManager<ExtraData = any> extends EventEmitter {
     public options: GiveawaysManagerOptions<ExtraData>;
     public ready: boolean;
 
+    protected generateMainEmbed(giveaway: Giveaway<ExtraData>, lastChanceEnabled?: boolean): EmbedBuilder;
+    protected generateEndEmbed(giveaway: Giveaway<ExtraData>, winners: GuildMember[]): EmbedBuilder;
+    protected generateNoValidParticipantsEndEmbed(giveaway: Giveaway<ExtraData>): EmbedBuilder;
     public delete(messageId: Snowflake, doNotDeleteMessage?: boolean): Promise<Giveaway<ExtraData>>;
     public deleteGiveaway(messageId: Snowflake): Promise<boolean>;
```

Comment:
```suggestion
    protected deleteGiveaway(messageId: Snowflake): Promise<boolean>;
```
but then maybe it should get moved down to the others?